### DISCUSSION TITLE
Very long render/simulation times?

### DISCUSSION THREAD

**Discussion by hydroculator** [inkstitch/inkstitch#1834] [Q&A]
I'm very new to this program, so I'm not sure what the expected behavior is.
I am attempting to use a Corel Draw file -> SVG -> Inkscape -> Inkstitch, and view it in the preview/simulator. 
It's a fairly complex drawing, and has so far taken an hour of spinning "Rendering stitch-plan". Is there a way to see if the program has crashed or given up? Is it likely to finish if it hasn't yet in an hour? I don't mind the waiting, I just want to be sure it's actually doing something.
Thanks

**Comment by tyrosinase** [comment]
It's probably waiting to show you an error, which it can't do until you close the GUI. It's a known problem, which will be fixed by either starting to use Electron behind the scenes or no longer using Electron behind the scenes, I forget which. :laughing: 

In any event, whenever you have a new, complex design, especially one created outside of Inkscape, you pretty much always want to run Troubleshoot on it before trying to render it, since that'll usually give you any errors right up front.

**Reply by schnoberts1** [reply]
I've also noticed slow performance with SVG's with thousands of points. I found simplifying the paths helped a lot.
